tests/test_command.py

```diff
@@ -5,7 +5,7 @@
     import unittest
 else:
     import unittest2 as unittest
-from mock import Mock
+from mock import *
```

Comment:
call function was not declared which is present in mock , so included * to solve all dependencies. Same thing is done on test_actions.py to solve this.